### PR TITLE
rework the proposal/control.xml description to fit current state

### PR DESCRIPTION
**Pull request by wfeldt**

### CODE REVIEW

File: doc/old_and_new_proposal.md [hunk 1]
```diff
@@ -142,39 +156,113 @@ into PVs and also added to the VG.
...
 
+#### First step
+
 A first step decides which volumes will be needed. Each volume will originate a
 partition or a LV in the second step.
 
-If the new format of `control.xml` is used (see [the specification
-document](old_and_new_control.md)), the basic list of planned volumes will be
-taken from the `<volumes>` subsection of the control file, except for those
+If the new format is used in the `partitioning` section in`control.xml`
```
Comment (joseivanlopez) [inline doc/old_and_new_proposal.md]: Space missing "in`control.xml`"

File: doc/old_and_new_proposal.md [hunk 2]
```diff
@@ -441,9 +531,13 @@ proposal to:
     proposal does),
   * never propose separate `/home` or swap partitions,
   * propose a separate `/var/lib/docker` partition if there is enough space for
-    it (basically if the disk is bigger than 20GiB, according to the sizes in
+    it (basically if the disk is bigger than 20 GiB, according to the sizes in
     the example).
 
+> FIXME:
+> `proposal_settings_editable`, `expert_partitioner_warning`, `root_subvolume_read_only`
+> in the following example do not exist (yet). Equivalents would have to appeasr in the `proposal` section.
```
Comment (joseivanlopez) [inline doc/old_and_new_proposal.md]: Typo: appeasr

File: doc/old_and_new_proposal.md [hunk 3]
```diff
@@ -644,6 +768,10 @@ calculated. If YaST is not able to make space for all the volumes required by
 those default settings, it will perform new attempts altering the settings. For
 that, it will follow the `disable_order` for each volume with that field.
 
+> FIXME: unclear (and not yet implemented anyway)
+>
+> `adjust_by_ram` is optional == `adjust_by_ram_configurable`?
+
```
Comment (joseivanlopez) [inline doc/old_and_new_proposal.md]: Yes, optional here stands for configurable. This is implemented in the PR that is being reviewed by @imobachgs : https://github.com/yast/yast-storage-ng/pull/384

File: doc/old_and_new_proposal.md [hunk 4]
```diff
@@ -768,12 +768,8 @@ calculated. If YaST is not able to make space for all the volumes required by
 those default settings, it will perform new attempts altering the settings. For
 that, it will follow the `disable_order` for each volume with that field.
 
-> FIXME: unclear (and not yet implemented anyway)
->
-> `adjust_by_ram` is optional == `adjust_by_ram_configurable`?
-
 In the first iteration, it will look for the lowest number there. If
-`adjust_by_ram` is optional in that volume and enabled, it will disable it. If
+`adjust_by_ram_configurable` is true in that volume, it will disable `adjust_by_ram`. If
```
Comment (joseivanlopez) [inline doc/old_and_new_proposal.md]: NP: well, it will disable it only in case `adjust_by_ram` is true.

Comment (wfeldt) [reply doc/old_and_new_proposal.md]: Is there a difference, technically?

Comment (joseivanlopez) [reply doc/old_and_new_proposal.md]: Not, really

**Pull request merged**